attribution_scopes.md

```diff
@@ -0,0 +1,229 @@
+
+### Updating attribution scope values
+
+An API caller may want to update the value of `attribution_scope_limit` for certain registrations or at a certain time. For example, when an advertiser starts a new campaign the API caller may want to increase the value of `attribution_scope_limit` to account for this new campaign.
+
+The `attribution_scope_limit` value can be updated during source registration at any time. However, any pending sources (previously registered) that have been specified with an `attribution_scope_limit` less than the current source registration’s `attribution_scope_limit` will be deleted.
+
+### Deletion logic
+
+If the current trigger is attributed to a source, then all other eligible sources considered for attribution (across all `attribution_scopes`) will be deleted.
+
+## Attribution Scope Examples
+
+### Example 1: distinct attribution scopes
+
+This example shows an API caller that manages 3 advertisers that all sell products on the same destination site (scheme + eTLD+1). In this example the API caller uses an `attribution_scope_limit` of 3 and tracks each advertiser with a distinct `attribution_scopes` value. Additionally, the user browsing the web sees 4 different ads from these advertisers and each one has a source registration associated:
+
+```jsonc
+// source registration 1 for advertiser1 at t=0
+{
+  ..., // existing fields
+  "attribution_scope_limit": 3,
+  "attribution_scopes": ["advertiser1"],
+  "max_event_states": 3
+}
+```
+
+```jsonc
+// source registration 2 for advertiser1 at t=1
+{
+  ..., // existing fields
+  "attribution_scope_limit": 3,
+  "attribution_scopes": ["advertiser1"],
+  "max_event_states": 3
+}
+```
+
+```jsonc
+// source registration 3 for advertiser2 at t=2
+{
+  ..., // existing fields
+  "attribution_scope_limit": 3,
+  "attribution_scopes": ["advertiser2"],
+  "max_event_states": 3
+}
+```
+```jsonc
+// source registration 4 for advertiser3 at t=3
+{
+  ..., // existing fields
+  "attribution_scope_limit": 3,
+  "attribution_scopes": ["advertiser3"],
+  "max_num_event_states": 3
+}
+```
+
+The user then converts at a later time on the destination site by purchasing a product associated with advertiser1:
+
+```jsonc
+// trigger registration 1 for advertiser1 at t=4
+{
+  ..., // existing fields
+  "attribution_scopes": ["advertiser1"]
+}
+```
+
+The API automatically performs attribution between any sources that have `attribution_scopes` that are not disjoint with the trigger `attribution_scopes`. Any sources that do not have an `attribution_scopes` that matches at least one of the trigger registration `attribution_scopes` are deleted (assuming there is a source that gets attributed and a report is generated; if no report is generated then no sources are deleted). In this example, the API caller would receive an attribution report attributing the trigger registration to advertiser1’s second source registration.
```

Comment:
"...if no report is generated then no sources are deleted..." is this true? Or is it enough that top-level filters match?